main/src/cgeo/geocaching/AbstractDialogFragment.java

```diff
@@ -220,7 +220,8 @@ protected final void addCacheDetails() {
             final int findsCount = cache.getFindsCount();
             if (findsCount > 0) {
                 details.add(R.string.cache_favorite, res.getString(R.string.favorite_count_percent, favCount, (float) (favCount * 100) / findsCount));
-            } else {
+            }
+            else if (!cache.isEventCache()) {
```

Comment:
please place it on one row
~~~
            } else if (!cache.isEventCache()) {
~~~

Reply:
My fault, it's fixed now.